mutatingwebhook_test.go

```diff
@@ -132,10 +180,69 @@ func TestReadyEndpoint(t *testing.T) {
 	assert.Equal(t, "ok", bodyString)
 }
 
+func writeCerts(certDir, name string) error {
+	certFile := filepath.Join(certDir, "tls.cert")
+	keyFile := filepath.Join(certDir, "tls.key")
+
+	cert := &x509.Certificate{
+		SerialNumber: big.NewInt(2019),
+		Subject: pkix.Name{
+			Organization:  []string{"Statistics Canada"},
+			Country:       []string{"CA"},
+			Province:      []string{"ON"},
```

Comment:
Minor nitpick, the province should be fully spelt out in certificates.

Reply:
Good to know! Thanks!

Reply:
Fixed!